projects/m1/031-coin-flip-simulation/solutions/alessiacaporaso/coin-flip-simulation.js

```diff
@@ -0,0 +1,28 @@
+let flipSimulation;
+let flips;
+let averageFlips = 0;
+
+// program is run it should perform the simulation 10 times
+for (let i = 0; i <= 10; i++) {
+  let coin = [];
```

Comment:
Here you can use `const` ;-)
With primitive data, if you use const you store a constant value, with arrays you store the constant memory address where it is stored.
https://www.w3schools.com/js/js_array_const.asp